front/lib/validators/user.js

```diff
@@ -0,0 +1,10 @@
+import { email, firstName, lastName, password } from "@@/lib/constants/labels"
+import { string } from "yup"
+
+export const firstNameValidator = string().min(2).max(255).label(firstName)
+export const lastNameValidator = string().min(2).max(255).label(lastName)
+export const emailValidator = string().email().label(email)
+export const passwordValidator = string()
```

Comment:
Can this error be translated ?
It should also have been moved to the errors.js file

Reply:
Will move this in an issue to be thought about later on.

Reply:
Agree with it, it can be done later